crates/cli/src/lib.rs

```diff
@@ -25,13 +25,24 @@ use app::{App, Commands, DockerCommands, MigrateCommands, NodeCommands, QueryCom
 use clap::Parser;
 use console::Term;
 use enums::{CacheMode, LogLevel};
+use lenient_semver::Version;
 use moon_logger::{color, debug, LevelFilter, Logger};
 use moon_terminal::ExtendedTerm;
+use serde::{Deserialize, Serialize};
 use std::env;
+use std::error::Error;
 use std::path::PathBuf;
+use tokio::task;
 
 pub use app::BIN_NAME;
 
+const CURRENT_VERSION_URL: &str = "https://launch.moonrepo.app/versions/cli/current";
```

Comment:
maybe move the launchpad code to the `moonbase` crate, like `moonbase/src/launchpad.rs`?